src/materials/ConcreteASREigenstrain.C

```diff
@@ -267,7 +267,7 @@ ConcreteASREigenstrain::computeResidual(unsigned qp, Real scalar)
     {
       f = 1.0 + _alpha * I_sigma / (3.0 * -_f_compress);
       if (f < 1.0)
-        mooseError("Wrong retardation for ASR latency time calculation!");
+        mooseAssert("f < 1.0", "Wrong retardation for ASR latency time calculation!");
```

Comment:
Just checking -- it's not actually possible to hit this condition, is it?

Also, line 269 should be deleted because it's redundant with the assertion. Also, the assertion should read `"f >= 1.0"`.

Reply:
Yes, it is not possible to hit, that is why I changed "mooseError" to mooseAssert. I will correct it.